edge-hub/docker/linux/amd64/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN apk update && \
     apk add --no-cache snappy libcap
 
 # Add an unprivileged user account for running Edge Hub
-RUN adduser -Ds /bin/sh edgehubuser 
+ARG EDGEHUBUSER_ID=1000
```

Comment:
Why is this not needed for linux/arm?

Reply:
It is, Angelo build the base images so he also took the change for the uid